src/nomparser/grammar.ebnf

```diff
@@ -0,0 +1,38 @@
+
+newline = "\n";
```

Comment:
newline is different on different platform

Reply:
updated based on https://stackoverflow.com/questions/3720612/what-is-the-universal-newline-for-all-operating-systems-lf-and-cr